docs/configuration.md

```diff
@@ -119,7 +119,7 @@ datafeed:
     maxIntervalMillis: 10000
 
 retry:
-  maxAttempts: 6
+  maxAttempts: 6 # set '-1' for an infinite number of attempts, default value is '10'
```

Comment:
default is infinite isn't it?

Reply:
For datafeed retry default is `-1`, otherwise for global retry we kept `10`. Is it something we should align? I was just thinking that an infinite retry is not something that we would expect when calling regular endpoints

Reply:
my bad I thought it was for the DF loop only